### PR TITLE
Add a Container component

### PR DESCRIPTION
**Pull request by evadb**
This PR adds a general use styled div called Container, used to uh, contain... things. I've added the props necessary to handle my upcoming Top Header component, mostly flexbox and basic height/width things. We will likely need to add onto this in the future for more complex layouting, but this gives us a start. Also, check out them tests ;)

### CODE REVIEW

File: frontend/src/components/container/__tests__/Container.test.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    expect(test).toHaveStyleRule('align-content', 'left')
+    expect(test).toHaveStyleRule('justify-content', 'right')
+    expect(test).toHaveStyleRule('flex-wrap', 'wrap')
+    expect(test).toHaveStyleRule('flex-basis', 'auto')
+    expect(test).toHaveStyleRule('flex-direction', 'column')
+    expect(test).toHaveStyleRule('flex', '0 0 auto')
+    expect(test).toHaveStyleRule('justify-self', 'center')
+    expect(test).toHaveStyleRule('align-self', 'center')
+  })
+})
```
Comment (sastels) [inline frontend/src/components/container/__tests__/Container.test.js]: 👏

File: frontend/src/components/container/__tests__/Container.test.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { ThemeProvider } from 'emotion-theming'
+import { Container } from '../'
+import { render, cleanup } from 'react-testing-library'
+
+describe('<Container />', () => {
+  afterEach(cleanup)
+  // const example = 'example'
```
Comment (sastels) [inline frontend/src/components/container/__tests__/Container.test.js]: 🔥

Comment (evadb) [reply frontend/src/components/container/__tests__/Container.test.js]: 🔥 🔥

**Comment by sastels** [comment]
[Build basic components](https://trello.com/c/2j6bZwVU/178-build-basic-components)

**Pull request merged**